src/r-apt/install.sh

```diff
@@ -123,8 +126,20 @@ Pin: release l=CRAN-Apt Packages
 Pin-Priority: 700
 EOF
 elif [ "${ID}" = "debian" ]; then
-    curl -fsSL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x95c0faf38db3ccad0c080a7bdc78b2ddeabc47b7" | tee -a /etc/apt/trusted.gpg.d/cran_debian_key.asc
-    echo "deb [arch=amd64] http://cloud.r-project.org/bin/linux/debian ${VERSION_CODENAME}-cran40/" >/etc/apt/sources.list.d/cran-debian.list
+    if [ "${USE_UNSTABLE}" = "true" ]; then
+        echo "Set up Debian unstable..."
+        echo "deb http://http.debian.net/debian sid main" >/etc/apt/sources.list.d/debian-unstable.list
+        # Pinning
+        cat <<EOF >"/etc/apt/preferences.d/99debian-unstable"
+Package: *
+Pin: release a=unstable
+Pin-Priority: 700
+EOF
```

Comment:
@eddelbuettel Could you please see if this is a reasonable way to set up unstable?

Reply:
a) I wouldn't, and I don't see.  The rocker/r-base aka r-base -- it just blends _testing_ (much more stable than unstable) with access to unstable if/when you need (sometimes packages are 'stuck' in unstable during transitions

b) I don't think you need a pin and here too I would recommend against.  Now,  I may be missing something but did you actually run into an issue?

Reply:
Thank you for your comment.
I actually don't use this and have not encountered any problems.
I only tried installing R on a Debian container with unstable set up as a test.

Do you think testing (and making it the default setting) makes more sense than unstable?

Reply:
Yes.  It is what I and many others (CRAN Vienna among them) have used for many years, and what our r-base container has done for many years.

Reply:
Thanks, I definitely do not fully understand the difference between unstable and testing!
It certainly seems to make sense to set testing to install the latest R packages.
(It installed dplyr 1.0.10 which is what I wanted to use!)